site/en/user_guide.md

```diff
@@ -16,36 +16,27 @@ setup guide, you can either use the [OCR-D-Docker-solution](https://ocr-d.github
 started with OCR-D after the installation as detailed in the very next two paragraphs. The [third preparatory step](#preparing-a-workspace) is
 obligatory for both Docker and Non-Docker users!
 
-Furthermore, Docker commands have a [different syntax than native calls](#translating-native-commands-to-docker-calls). This guide always states native calls first and then provides the respective command for Docker users.
+Furthermore, Docker commands have a [different syntax than native calls](#translating-native-commands-to-docker-calls). 
+This guide always states native calls first and then provides the respective command for Docker users.
 
-## Prerequisites and Preparations
+## Preparations
 
-### Setup docker
+### Docker installation: 
 
-If you want to use the OCR-D-Docker-solution, [Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-using-the-repository) and [docker compose](https://docs.docker.com/compose/install/) have to be installed.
-
-After installing docker you have to set up daemon and add user to  the group 'docker'
-
-```sh
-# Start docker daemon at startup
-sudo systemctl enable docker
-# Add user to group 'docker'
-sudo usermod -aG docker $USER
-```
-
-<img src="https://github.githubassets.com/images/icons/emoji/unicode/26a0.png" alt="warning" style="zoom:33%;" /> Please log out and log in again.
-
-To test access to docker try the following command:
+If you are using the Installation via Docker, we recommend to run:
 
 ```sh
-docker images
+docker run --user $(id -u) --workdir /data --volume $PWD:/data --volume $PWD/models:/usr/local/share/ocrd-resources --volume $PWD/models:/usr/local/share/tessdata --volume $PWD/models:/usr/local/share/ocrd-resources -it ocrd/all bash
```

Comment:
See above wrt. to tesseract location in `$PWD/models`.

Reply:
Do you mean the folder _ocrd-tesserorcr-recognize_ possibly not existing in `models`?

Reply:
I meant that `$PWD/models:/usr/local/share/tessdata` should be `$PWD/models/ocrd-tesserocr-recognize:/usr/local/share/tessdata`. The folder will be created on the host if it does not exist, so the `mkdir -p` is not strictly necessary but as I said, I think that's cleaner to make this explicitly on the host.